uptane-standard.md

```diff
@@ -214,10 +214,14 @@ These terms are defined in greater detail in {{roles}}.
 
 ## Acronyms and abbreviations
 
+*CAN Bus*: Controller Area Network bus standard
+
 *CDN*: Content Delivery Network
 
 *ECUs*: Electronic Control Units, the computing units on a vehicle
 
+*LIN Bus*: Local Interconnect Bus
```

Comment:
This appears to be reverting https://github.com/uptane/uptane-standard/pull/191. I think we can safely remove the CAN reference, but LIN is mentioned below, so we probably should restore that.

Reply:
It's odd that we would define LIN but not CAN, given how prevalent the latter is. Do we really need it?

Reply:
I was wondering that, too.

FWIW, it looks like `CAN` was *never* used. The abbreviation was added early on in 48df2810120d9f29482c55f6b3c89a843d71853a, presumably copied from somewhere else or under the assumption it was inevitably going to get used.

Reply:
I put together the list of definitions and I can confirm that most of the terms were copied, either from earlier versions of one of the documents, or the slide presentations. It could be I originally added both because I anticipated we would need both.  Perhaps for this audience its overly simplistic to mention either, but it is a general rule of thumb to define an acronym before using it. I think we can removed both if LIN is defined in the text where it is first used.

Reply:
LIN is not spelled out elsewhere in the text, so I think if we keep it as is, we should restore the definition. CAN can definitely go.